src/content/docs/features/endpoints.md

```diff
@@ -2,15 +2,9 @@
 title: HTTP Endpoints
 ---
 
-<!--
-TODO: Show the following:
+<!-- TODO: add how to programmatically get the endpoint -->
```

Comment:
I can't remember — how do you access a program's endpoint?

Reply:
`await nodes.process.endpointUrl`

Reply:
And is there a way to access the endpoint URL of a connection/dependency programmatically?

Reply:
Did you have a use-case for something like that in mind?

Reply:
I think I wanted to link to the [`membrane-getting-started-feedback-form`](https://github.com/membrane-io/membrane-getting-started-feedback-form) from the [`membrane-getting-started`](https://github.com/membrane-io/membrane-getting-started) program, but hardcoding the link is easy enough.